gensrc/proto/internal_service.proto

```diff
@@ -580,6 +580,8 @@ message PExecShortCircuitResult {
 }
 
 message PExecShortCircuitRequest {
+    // describe attachment protocol: binary, compact, json
+    optional string attachment_protocol = 1;
```

Comment:
Could you use Enum type rather than string type?

Reply:
refer to https://github.com/StarRocks/starrocks/pull/29888/files

if use Enum or int, it will cause compatable issues during upgrading, as the new added field will set the default value 0, corresponding to `compact` in `TProtocolType`, not the expected `binary`.